src/Process/PointCloudProcessing.js

```diff
@@ -0,0 +1,291 @@
+import * as THREE from 'three';
+import convexHull from 'monotone-convex-hull-2d';
+
+function cube(size) {
+    var h = size.clone().multiplyScalar(0.5);
+    var geometry = new THREE.Geometry();
+    geometry.vertices.push(
+        new THREE.Vector3(-h.x, -h.y, -h.z),
+        new THREE.Vector3(-h.x, h.y, -h.z),
+        new THREE.Vector3(-h.x, h.y, -h.z),
+        new THREE.Vector3(h.x, h.y, -h.z),
+        new THREE.Vector3(h.x, h.y, -h.z),
+        new THREE.Vector3(h.x, -h.y, -h.z),
+        new THREE.Vector3(h.x, -h.y, -h.z),
+        new THREE.Vector3(-h.x, -h.y, -h.z),
+        new THREE.Vector3(-h.x, -h.y, h.z),
+        new THREE.Vector3(-h.x, h.y, h.z),
+        new THREE.Vector3(-h.x, h.y, h.z),
+        new THREE.Vector3(h.x, h.y, h.z),
+        new THREE.Vector3(h.x, h.y, h.z),
+        new THREE.Vector3(h.x, -h.y, h.z),
+        new THREE.Vector3(h.x, -h.y, h.z),
+        new THREE.Vector3(-h.x, -h.y, h.z),
+        new THREE.Vector3(-h.x, -h.y, -h.z),
+        new THREE.Vector3(-h.x, -h.y, h.z),
+        new THREE.Vector3(-h.x, h.y, -h.z),
+        new THREE.Vector3(-h.x, h.y, h.z),
+        new THREE.Vector3(h.x, h.y, -h.z),
+        new THREE.Vector3(h.x, h.y, h.z),
+        new THREE.Vector3(h.x, -h.y, -h.z),
+        new THREE.Vector3(h.x, -h.y, h.z));
```

Comment:
I know but `OBBHelper` is designed around `OBB` so it won't work for my use-case without some refactoring.